example/grpc-example/README.md

```diff
@@ -58,24 +55,20 @@ Steps:
 The following is the output you'll see from the server and client side after running the previous commands.
 **Server**
 ```
-$ ./grpc-server/server --identity grpc.server.json --service grpc
+$ ./grpc-server --identity grpc.server.json --service grpc
 2022/10/21 11:17:34 server listening at grpc
 2022/10/21 11:18:09 Received: World
 ```
 **Client**
 ```
-$ ./grpc-client/client --identity grpc.client.json --service grpc --name geoff
+$ ./grpc-client --identity grpc.client.json --service grpc --name World
```

Comment:
World --> grpc.client ?

Reply:
This was a copy/paste issue where I actually pasted the output of me entering the name as `geoff` but the example uses `World` so that it replies back with "Hello World"

Reply:
lol - oh i see... 👍